### ISSUE TITLE
Synced surveys not saving telephone number

### ISSUE THREAD

**Issue by aprojas** [FundacionParaguaya/MentorApp#1082]
Hub: Emprender
Organization: Congente (maybe others)
Facilitator users: to be confirmed
Connection: Offline

Problem
---
The Hub Emprender is reporting that some of their mentors that are taking surveys, add the telephone number of the family, but the telephone data is not being sent to the reports (or database). 
So, after the surveys are synced, they cannot see it in the reports. 

These surveys were taken this week and the previous one.

**Comment by Danman7** [comment]
Surveys tested: both Emprender surveys, Nelixia, Paraguay Fupa, Geco and Salvation Army. We used all sorts of formats for the phone numbers: '32432315', '32 2345 234', '235-234-234'. In the 9 surveys we took, not once did the phone number deleted itself in the app or was not passed to the server with the API call. Also, every single family we saved showed the phone in the review screen, so it seems the server also works.

This means we cannot reproduce the issue with our testing methods, be it in the app or the server. I'm pausing this issue, without closing it.
